src/main/webapp/view/transcript/transcript-area.js

```diff
@@ -21,9 +21,12 @@ export default class TranscriptArea {
   static #TRANSCRIPT_CONTAINER = 'transcript-lines-container';
   static #PARAM_ID = 'id';
 
+  static transcriptLineMap = new Map();
```

Comment:
Is there a reason this isn't private?

Reply:
Ah, there isn't. I updated it to be private, thanks!